src/components/Book.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+const Book = ({ title, author, item_id: ïtemId }) => {
+  const dispatch = useDispatch();
+
+  const handleRemove = (e) => {
+    e.prevetDefault();
+    dispatch(removeBook(ïtemId));
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleRemove}>
+        <div>
+          <p>{title}</p>
+          <p>{author}</p>
+        </div>
+        <button type="submit">
+          Remove
+        </button>
+      </form>
+    </div>
+  );
```

Comment:
- [ ] A very good job so far :clap: , However, am not able to remove any book at the moment because of the following mistakes: Your `button` type should be `button` not `submit` and this line of code  `onSubmit={handleRemove}` should be in button not within the `form` tag and your event handler should be `onClick` event not `onSubmit`.
- [ ] Please remove also `e.prevetDefault()` :slightly_smiling_face: 

The possible solution would be: :arrow_down: 

`const handleRemove = () => {`
   ` dispatch(removeBook(ïtemId));`
 ` };`

`  return (`
    `<div>`
   `   <form>`
     `   <div>`
        `  <p>{title}</p>`
       `   <p>{author}</p>`
      `  </div>`
   `     <button onClick={handleRemove} type="button">`
         ` Remove`
  `      </button>`
    `  </form>`
`    </div>`
`  );`